frontend/src/App.js

```diff
@@ -157,9 +157,17 @@ function App() {
 
       <Route exact path="/" element={<ProtectedRoute isAuth={isAuth} setIsAuth={setIsAuth} />}>
         {/* Profile Page */}
+        <Route
```

Comment:
I can't make a multiline comment since some of what I wanted to select isn't new, but in order to use react-router's `useParam()` hook we need to route like this:

```jsx
{/* Profile Page */}
<Route
    path={SITE_PAGES.PROFILE}
    element={
        <PageLayout>
            <Outlet />
        </PageLayout>
    }
>
    <Route
        path=":id"
        element={<ProfilePage />}
    />
    <Route
        path={SITE_PAGES.PROFILE}
        element={<ProfilePage />}
    />
</Route>
```